lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb

```diff
@@ -103,7 +104,7 @@ def self.wait_for_test_results(ftl_service, gcp_project, matrix_id, async)
 
               if async
                 UI.success("Job(s) have been submitted to Firebase Test Lab")
-                return
+                return firebase_console_link
```

Comment:
Why making this change? Don't see how it's been used.

Reply:
We want the firebase console link. We post to slack after the job has been successful